src/EmailAddress.php

```diff
@@ -342,6 +344,8 @@ protected function validateLocalPart()
         $atext = 'a-zA-Z0-9\x21\x23\x24\x25\x26\x27\x2a\x2b\x2d\x2f\x3d\x3f\x5e\x5f\x60\x7b\x7c\x7d\x7e';
         if (preg_match('/^[' . $atext . ']+(\x2e+[' . $atext . ']+)*$/', $this->localPart)) {
             $result = true;
+        } elseif ($this->validateInternationalizedLocalPart()) {
```

Comment:
To make this a bit more robust, please pass `$this->localPart` as an argument to `validateInternationalizedLocalPart()`. When we make the validators stateless for v3, this will then be one less method we need to change. :smile: